### ISSUE TITLE
Coop leaderboards are broken: sim modded games counts 

### ISSUE THREAD

**Issue by K-ETFreeman** [FAForever/server#913]
Just check out the  # 1  replay of 5th mission (Mind Games, 23m 40s by pobus)
You'll see this: 
![image](https://user-images.githubusercontent.com/55841348/169397548-29cbc2fa-c355-4090-b4f7-96a0f66da392.png)

This guy obvsiously have sim mod (resourses x1000000 or something like that)
So leadebord has no meaning because of this bug, need to fix & clear leaderboard from modded replays somehow

**Comment by K-ETFreeman** [comment]
Now meltdown (4th mission) is taken by the cheater too
and 1st and 2nd places in Mind games (5th mission) is taken by the cheaters
need to do something :C

**Comment by Brutus5000** [comment]
Note: We need a new bantype coop

**Comment by K-ETFreeman** [comment]
Meltdown mission is not taken by cheater too (the guy with nickname "momoception")
replay id is 17992592

**Comment by Askaholic** [comment]
So I just took a look at the replay file and there are no sim mods being used. I also checked the server code and it should be invalidating any games that are using sim mods. So my guess is that this player is using some sort of exploit in game. That's unfortunately nothing we can detect on the server side so I'm gonna have to close this issue. Maybe @Garanas wants to take a look at the replays though in case there's a game bug that can be fixed.
